common-app-lib/test/cmr/common_app/test/api/health.clj

```diff
@@ -30,21 +31,22 @@
         unhealthy-response {:status 503
                             :body (json/generate-string
                                    {:test-resource {:ok? false
-                                                    :problem "Resource is not responding."}})}]
+                                                    :problem "Resource is not responding."}})}
+        context-with-health-cache (get-context-with-health-cache)]
     (is (= healthy-response
            (select-keys (get-app-health context-with-health-cache healthy-fn) [:status :body])))
     (testing "health response is cached so even though the unhealthy-fn is passed in it is not used"
       (is (= healthy-response
              (select-keys (get-app-health context-with-health-cache unhealthy-fn)
                           [:status :body])))
       ;; Wait 2 seconds and the cache has not expired, so it still returns healthy
-      (Thread/sleep 2000)
+      (Thread/sleep 2001)
```

Comment:
I was seeing some inconsistent results over a few runs, so I just made sure that the time from making the first call to the final call was guaranteed to be more than 5 seconds.